index.js

```diff
@@ -43,6 +43,74 @@ exports.Store = Store;
 exports.Cookie = Cookie;
 exports.Session = Session;
 exports.MemoryStore = MemoryStore;
+exports.sessionId = {
+  /**
+   * Set sessionId to cookie.
+   *
+   * @private
+   */
+  set: function (res, name, sessionId, secret, options) {
+    var signed = 's:' + signature.sign(sessionId, secret);
+    var data = cookie.serialize(name, signed, options);
+
+    debug('set-cookie %s', data);
+
+
+    var prev = res.getHeader('set-cookie');
+    var header = [];
+
+    if (prev) {
+      header = header.concat(prev);
+    }
+
+    res.setHeader('set-cookie', header.concat(data))
+  },
+
+  /**
+   * Get the session ID cookie from request.
+   *
+   * @return {string}
+   * @private
+   */
+  get: function (req, name, secrets) {
```

Comment:
Just reference the old `getcookie` function instead of copy and pasting it up here (where it ends up with another level of indentation).